statsd-exporter/include/mappings-gen2.yml

```diff
@@ -268,6 +268,90 @@ mappings:
       instance: "$1"
       mount_path: "$2"
 
+  # ------------------------------------------------------------
```

Comment:
This is relying heavily on `([^.]+)` which is fine if you control sanitisation, but brittle otherwise.